thin_client_test.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import thin_client
+import socket
+from wheel.signatures import assertTrue
+            
+class thin_client_test(unittest.TestCase):
+    # Check if packAndSend increments the sequence number correctly       
+    def test_packAndSend_sequence(self):
+        sequence = 0
+        socketName = socket.socket(socket.AF_INET, # Internet
+                                   socket.SOCK_DGRAM) # UDP
+        finalSequence = thin_client.packAndSend(1, sequence, 0, 0, 0, 0, socketName)
```

Comment:
in test you may want to use Arrange Act Assert pattern. whereby each test, does this.